### PR TITLE
Fixes add links in the side navigation bar to the tutorial pages, make the title and sequence of the topics consistent

### LINKED ISSUE #11933: [UI] [Bug] Add links in the side navigation bar to the tutorial pages, make the title and sequence of the topics consistent
### Current Behavior
In the section of Guides & Tutorials in docs.meshery.io, there're several pages lacking link in the side navigation bar:

doc1

doc2

doc3

Furthermore, some of the titles listed in the navigation bar is not consistent with the page's title. For example, in the Configuring Kubernetes Infrastructure section, it's listed as "Working with Designs" in the navigation bar, while named "Configuring your Cloud Native Infrastructure and Applications" on the page content.

The sequence of the title appeared in the navigation bar is also not consistent with that listed on the page.

This issue is similar to #11930, but with more tasks.

### Desired Behavior
1. Add links in the navigation bar:
   - link to [Envoy WASM Filter Management](https://docs.meshery.io/guides/configuration-management/filter-management), and [Push or Pull a Model Image](https://docs.meshery.io/guides/configuration-management/push-pull-model-image) in Configuring Kubernetes Infrastructure section
   - link to [Mesheryctl system commands](https://docs.meshery.io/guides/mesheryctl/system-commands), and [Upgrading Meshery CLI](https://docs.meshery.io/installation/upgrades#upgrading-mesheryctl-using-homebrew) in Meshery CLI Guides
2. Make the title in the navigation bar consistent with that is listed on the page content in Configuring Kubernetes Infrastructure,Meshery CLI guides, and Performance Management
3. Make the sequence of titles in the navigation bar consistent with that is listed on the page in Configuring Kubernetes Infrastructure,Meshery CLI guides, and Performance Management
4. Fix the problem mentioned in the [comment](https://github.com/meshery/meshery/issues/11933#issuecomment-2337155494) below by @Ashparshp. Make sure that when click on a sidebar link, it will be highlighted and show the selection.
   The pages need to be fixed in Guides & Tutorials sections are,
   - [home / guides / infrastructure management / infrastructure discovery](https://docs.meshery.io/guides/infrastructure-management/registering-a-connection)
   - [home / guides / configuration management / creating a meshery design](https://docs.meshery.io/guides/configuration-management/creating-a-meshery-design)
   - make sure the newly added links in task 1 highlights when clicked

### Mockups

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Nandika-A**
**Notes for Reviewers**

This PR fixes #11933 

1. Added links to missing subtopics in Meshery CLI Guides and Configuring Kubernetes Infrastructure in the docs.
2. Made the title and the sequence in the navigation bar consistent with that is listed on the page in Configuring Kubernetes Infrastructure,Meshery CLI guides, and Performance Management.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 771dd1bf6a339a3b0fd45106524714d298802620 at: https://66e47f0e56c80c56357ec907--meshery-docs-preview.netlify.app

**Comment by Nandika-A** [comment]
Hi @leecalcote ! Could you please review this PR?

**Comment by l5io** [comment]
🚀 Preview for commit e52d7ee8d604f3cb2fe7cf6f9fc6655b7ff67a10 at: https://66e70443b957380ecd2d1ca5--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 4a68980f581b24fd7116ca2d3956e2597b08eebc at: https://66e727bab9573832212d1c80--meshery-docs-preview.netlify.app

**Comment by vishalvivekm** [comment]
@Nandika-A Thank you for your contribution! Let's discuss this during the website call on Monday at 5:30 PM IST (7:00 AM CT). Please consider adding it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk) || [meeting link](https://meet.layer5.io/websites).

// @Ashparshp

**Comment by Ashparshp** [comment]
@KunyueXing, would you mind taking a look at this when you have a moment?

File: docs/_data/toc.yml [hunk 1]
```diff
@@ -121,40 +121,48 @@
     - title: Configuring Kubernetes Infrastructure ↆ
       url: guides/configuration-management
       children:
+      - title: Configuring your Cloud Native Infrastructure and Applications
```
Comment (vishalvivekm) [inline docs/_data/toc.yml:124]:
```suggestion
      - title: Working with Designs
```

Comment (vishalvivekm) [reply docs/_data/toc.yml:124]: let's keep the original title, which also is shorter.

File: docs/_data/toc.yml [hunk 2]
```diff
@@ -121,40 +121,48 @@
...
       - title: Creating a Meshery Design
         url: guides/creating-a-meshery-design
-      - title: Working with Designs
-        url: guides/configuration-management/working-with-designs
+      - title: Envoy WASM Filter Management
+        url: guides/configuration-management/filter-management
       - title: Importing Designs
         url: guides/configuration-management/importing-designs
-      - title: Working with design patterns
+      - title: Cloud Native Design Patterns
```
Comment (vishalvivekm) [inline docs/_data/toc.yml:132]:
```suggestion
      - title: Working with design patterns
```

File: docs/_data/toc.yml [hunk 3]
```diff
@@ -121,40 +121,48 @@
...
       url: guides/mesheryctl
       children:
-        - title: Using mesheryctl
-          url: guides/mesheryctl/working-with-mesheryctl
-        - title: Configure autocompletion for mesheryctl
-          url: guides/mesheryctl/configuring-autocompletion-for-mesheryctl
-        - title: Authenticate via Meshery CLI
+        - title: Authenticating with Meshery via CLI
           url: guides/mesheryctl/authenticate-with-meshery-via-cli
+        - title: Configuring Autocompletion for `mesheryctl`
```
Comment (vishalvivekm) [inline docs/_data/toc.yml:156]:
```suggestion
        - title: Configuring Autocompletion for mesheryctl
```

File: docs/_data/toc.yml [hunk 4]
```diff
@@ -121,40 +121,48 @@
...
+          url: guides/mesheryctl/configuring-autocompletion-for-mesheryctl
         - title: Running system checks using Meshery CLI
           url: guides/mesheryctl/running-system-checks-using-mesheryctl
+        - title: Mesheryctl system commands
+          url: guides/mesheryctl/system-commands
+        - title: Using meshery CLI
+          url: guides/mesheryctl/working-with-mesheryctl
+        - title: Upgrading Meshery CLI
+          url: installation/upgrades#upgrading-mesheryctl-using-homebrew
```
Comment (vishalvivekm) [inline docs/_data/toc.yml:165]:
```suggestion
```

Comment (vishalvivekm) [reply docs/_data/toc.yml:165]: already in [Overview and Installation](https://66e727bab9573832212d1c80--meshery-docs-preview.netlify.app/installation/upgrades#upgrading-mesheryctl-using-homebrew)

File: docs/_data/toc.yml (hunk 2 above)
Comment (leecalcote) [reply docs/_data/toc.yml:132]: Please capitalize words in page titles.

**Comment by KunyueXing** [comment]
Hi @Ashparshp, of course, I’d be happy to take a look at this. 

I left a comment and it shows 'pending'. I'm not sure if the comment is only visible to me or it's public on this webpage.

**Comment by l5io** [comment]
🚀 Preview for commit 2b20f62d8fdff5ad9f9bc2be6afd808f9c959b38 at: https://66e95499a6799be9b232f43d--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 3bc9ce7971c1c78c6c02f368e411cf234280d0cd at: https://66e9556b95a9cfd673e3193c--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
> Hi @Ashparshp, of course, I’d be happy to take a look at this.
> 
> I left a comment and it shows 'pending'. I'm not sure if the comment is only visible to me or it's public on this webpage.

I'm not sure that it's visible until you submit the review with either "Comment", "Approve", or "Request Changes", @KunyueXing

File: docs/_data/toc.yml [hunk 5]
```diff
@@ -121,40 +121,46 @@
...
-        - title: Authenticate via Meshery CLI
+        - title: Authenticating with Meshery via CLI
           url: guides/mesheryctl/authenticate-with-meshery-via-cli
+        - title: Configuring Autocompletion for mesheryctl
+          url: guides/mesheryctl/configuring-autocompletion-for-mesheryctl
         - title: Running system checks using Meshery CLI
           url: guides/mesheryctl/running-system-checks-using-mesheryctl
+        - title: Mesheryctl system commands
+          url: guides/mesheryctl/system-commands
+        - title: Using meshery CLI
```
Comment (leecalcote) [inline docs/_data/toc.yml:162]: @Nandika-A, what do you think about capitalizing "Meshery" here?

**Pull request closed without merging**